views/dist/front.css.map

```diff
@@ -1 +1 @@
lay: inline-block;\n    border-width: 2px;\n    border-color: rgba(255, 255, 255, 0.05);\n    border-top-color: #fff;\n    animation: spin 1s infinite linear;\n    border-radius: 100%;\n    border-style: solid;\n  }\n}\n\n@keyframes spin {\n  100% {\n    transform: rotate(360deg);\n  }\n}\n","/**\n * Copyright since 2007 PrestaShop SA and Contributors\n * PrestaShop is an International Registered Trademark & Property of PrestaShop SA\n *\n * NOTICE OF LICENSE\n *\n * This source file is subject to the Academic Free License 3.0 (AFL-3.0)\n * that is bundled with this package in the file LICENSE.md.\n * It is also available through the world-wide-web at this URL:\n * https://opensource.org/licenses/AFL-3.0\n * If you did not receive a copy of the license and are unable to\n * obtain it through the world-wide-web, please send an email\n * to license@prestashop.com so we can send you a copy immediately.\n *\n * @author    PrestaShop SA <contact@prestashop.com>\n * @copyright Since 2007 PrestaShop SA and Contributors\n * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)\n */\n#search_filters {\n  .ui-slider-horizontal {\n    .ui-slider-handle {\n      margin-left: -1px;\n      cursor: pointer;\n    }\n  }\n  .ui-widget-header {\n    background: #555;\n  }\n  .ui-slider {\n    .ui-slider-handle {\n      top: -.45em;\n      width: 0.4em;\n      background: #fff;\n      border: 1px solid #555;\n    }\n  }\n  .ui-slider-horizontal {\n    height: .4em;\n  }\n}\n","/**\n * Copyright since 2007 PrestaShop SA and Contributors\n * PrestaShop is an International Registered Trademark & Property of PrestaShop SA\n *\n * NOTICE OF LICENSE\n *\n * This source file is subject to the Academic Free License 3.0 (AFL-3.0)\n * that is bundled with this package in the file LICENSE.md.\n * It is also available through the world-wide-web at this URL:\n * https://opensource.org/licenses/AFL-3.0\n * If you did not receive a copy of the license and are unable to\n * obtain it through the world-wide-web, please send an email\n * to license@prestashop.com so we can send you a copy immediately.\n *\n * @author    PrestaShop SA <contact@prestashop.com>\n * @copyright Since 2007 PrestaShop SA and Contributors\n * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)\n */\n@mixin text-ellipsis() {\n  width: calc(100% - 30px);\n  overflow: hidden;\n  text-overflow: ellipsis;\n  white-space: nowrap;\n}\n\n#search_filters {\n  .facet {\n    .title {\n      display: flex;\n      .collapse-icons {\n        margin-left: auto;\n      }\n    }\n\n    .facet-title {\n      @include text-ellipsis();\n    }\n\n    .facet-label {\n      width: 100%;\n      text-align: left;\n      .custom-checkbox,\n      .custom-radio {\n        top: -7px;\n        margin-right: 0;\n      }\n      .color {\n        margin-left: 0;\n      }\n\n      a {\n        @include text-ellipsis();\n      }\n    }\n  }\n}\n"],"names":[],"sourceRoot":""}
```

Comment:
what file modification did trigger this? :thinking:

Reply:
@matthieu-rolland Just running `npm install` and `npm run build`